app/serializers/cedar-metadata-record.ts

```diff
@@ -0,0 +1,10 @@
+import OsfSerializer from './osf-serializer';
+
+export default class CedarMetadataRecordSerializer extends OsfSerializer {
```

Comment:
For these cedar-related serializers, do we want to just use `JSONAPISerializer`?